Dockerfile.citus

```diff
@@ -42,7 +42,7 @@ RUN set -ex \
         && apt-get update -y \
         && apt-get -y install postgresql-$PG_MAJOR-citus-11.3; \
     fi \
-    && pip3 install dumb-init \
+    && apt-get install -y dumb-init \
```

Comment:
The same as https://github.com/zalando/patroni/pull/2770/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R29